Makefile

```diff
@@ -9,15 +9,8 @@ STACK_IMAGE_TAG := heroku/$(subst -,:,$(STACK))-build
 
 lint: lint-scripts lint-ruby
 
-# TODO: Enable scanning for files that are currently missed and/or restructure repo
-# layout to make it more viable to use wildcards here, given:
-# https://github.com/koalaman/shellcheck/issues/962
 lint-scripts:
-	@shellcheck -x bin/compile bin/detect bin/release bin/test-compile bin/utils bin/warnings bin/default_pythons
-	@shellcheck -x bin/steps/collectstatic bin/steps/nltk bin/steps/pip-install bin/steps/pipenv bin/steps/pipenv-python-version bin/steps/python
-	@shellcheck -x bin/steps/hooks/*
-	@shellcheck -x vendor/WEB_CONCURRENCY.sh
-	@shellcheck -x builds/*.sh
+	@git ls-files --cached --others --exclude-standard 'bin/*' '*/bin/*' '*.sh' | xargs shellcheck --check-sourced --color=always
```

Comment:
```suggestion
	@git ls-files -z --cached --others --exclude-standard 'bin/*' '*/bin/*' '*.sh' | xargs -0 shellcheck --check-sourced --color=always
```

Reply:
Given that ls-files quotes any unusual filenames, why is this needed?

Reply:
It doesn't quote for correct shell/xargs consumption:

```ShellSession
$ git ls-files foo*
"foo\nbar"
foo bar
$ git ls-files foo* | xargs cat
cat: foo\nbar: No such file or directory
cat: foo: No such file or directory
cat: bar: No such file or directory
```